source/_posts/2015-12-21-wye-tech-news-3.markdown

```diff
@@ -0,0 +1,84 @@
+---
+layout: post
+title: "Wye Tech News #3"
+category:
+comments: true
+author:
+  name: Jorge Bejar
+  email: jorge@wyeworks.com
+  twitter_handle: jmbejar
+  github_handle:  jmbejar
+  image:  /images/team/jorge-bejar.jpg
+  description: Software Engineer at Wyeworks. Ruby on Rails developer.
+---
+
+Hey it's Monday! So here is a new issue of our weekly compilation of interesting news and articles about Ruby, Rails, JavaScript and web development related stuff.
+
+<!--more-->
+
+
+**Ruby 2.3.0-preview2 Released**<br/>
+https://www.ruby-lang.org/en/news/2015/12/11/ruby-2-3-0-preview2-released/<br/>
+The most remarkable changes since preview1 are those to support a pre-compilation system and a few performance tuning fixes.
+
+**Ruby CVE-2015-7551: Unsafe tainted string usage in Fiddle and DL**<br/>
+https://www.ruby-lang.org/en/news/2015/12/16/unsafe-tainted-string-usage-in-fiddle-and-dl-cve-2015-7551/<br/>
+There is an unsafe tainted string usage vulnerability in Fiddle and DL. This vulnerability has been assigned the CVE identifier CVE-2015-7551.<br/>
+
+Ruby 2.2.4, Ruby 2.1.8 and Ruby 2.0.0-p648 were released including a security fix for Fiddle extension.<br/>
+https://www.ruby-lang.org/en/news/2015/12/16/ruby-2-2-4-released/<br/>
+https://www.ruby-lang.org/en/news/2015/12/16/ruby-2-1-8-released/<br/>
+https://www.ruby-lang.org/en/news/2015/12/16/ruby-2-0-0-p648-released/<br/>
+
+**Rails 5.0.0.beta1: Action Cable, API mode, Rails command**<br/>
+http://weblog.rubyonrails.org/2015/12/18/Rails-5-0-beta1/<br/>
+The first beta release of the new major version of Rails was released.<br/>
+
+In addition, you might be interested on how to upgrade a Rails app to
+Rails 5:<br/>
+http://edgeguides.rubyonrails.org/upgrading_ruby_on_rails.html#upgrading-from-rails-4-2-to-rails-5-0
+
+**Rails 5: Action Cable demo**<br/>
+https://www.youtube.com/watch?v=n0WUjGkDFS0&feature=youtu.be<br/>
+A quick demo of Action Cable in Rails 5 by building a simple chat application.
+
+**Rails Attributes API**<br/>
+https://github.com/rails/rails/blob/8c752c7ac739d5a86d4136ab1e9d0142c4041e58/activerecord/lib/active_record/attributes.rb<br/>
+A new `attribute` class method is available in Rails 5 models, allowing you to easily define a relationship between the model and a non-Active Record type.
+
+**ApplicationRecord**<br/>
+https://github.com/rails/rails/pull/22567<br/>
+`ApplicationRecord` model superclass in Rails 5
```

Comment:
All models inherit from `ApplicationRecord` in Rails 5, this means no more monkey patches of `ActiveRecord::Base`.